etc/cvmfs/domain.d/lsst.eu.conf

```diff
@@ -1,10 +1,17 @@
-# Replace this file with a symlink to ../osghosted.conf once the selinux
-#  issue in https://sft.its.cern.ch/jira/browse/CVM-1922 is no longer a
-#  concern.
-
-# refer to all variables set by osghosted.conf so cvmfs will read them
+# These are here so cvmfs will notice them if common.conf sets them
 CVMFS_USE_CDN="$CVMFS_USE_CDN"
 CVMFS_HTTP_PROXY="$CVMFS_HTTP_PROXY"
 CVMFS_FALLBACK_PROXY="$CVMFS_FALLBACK_PROXY"
-CVMFS_SERVER_URL="$CVMFS_FALLBACK_PROXY"
-. ../osghosted.conf
+. ../common.conf
+
+if [ "$CVMFS_USE_CDN" = "yes" ]; then
+    CVMFS_SERVER_URL="http://s1cern-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1in2p3lsst-cvmfs.openhtc.io/cvmfs/@fqrn@"
+    CVMFS_SERVER_URL="$CVMFS_SERVER_URL;http://s1bnl-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1fnal-cvmfs.openhtc.io/cvmfs/@fqrn@"
+else
+    CVMFS_SERVER_URL="http://cvmfs-stratum-one.cern.ch:8000/cvmfs/@fqrn@;http://cclssts1.in2p3.fr/cvmfs/@fqrn@"
+    CVMFS_SERVER_URL="$CVMFS_SERVER_URL;http://cvmfs-s1bnl.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1fnal.opensciencegrid.org:8000/cvmfs/@fqrn@"
+fi
+CVMFS_PUBLIC_KEY="$CVMFS_MOUNT_DIR/$CVMFS_CONFIG_REPOSITORY/etc/cvmfs/keys/lsst.eu.pub"
```

Comment:
Shouldn't we create a domain directory for the key?

Reply:
It wasn't that way in egi, and I think it's best to keep it consistent across branches.  I don't see much value added when there's only a single key.  If a second one is needed later we can always change it.